management/src/commands/update-identity-from-user.ts

```diff
@@ -53,7 +53,7 @@ async function updateIdentityFromUser(
     const identity = {
       userId,
       identityType: getIdentityType(identityToUpdate.type),
-      identityValue: identityToUpdate.value,
+      identityValue,
```

Comment:
Nope, because in the first place, we want to delete the "old" identity, and not the new one. If this step fails, then we want to delete the "new" identity we just added to rollback in initial state.
Here `identityValue` is the value of the "new" identity we want to replace the old with.